control/control.go

```diff
@@ -623,14 +623,12 @@ func (c *Controller) gc() {
 	}()
 
 	for _, w := range workers {
-		func(w worker.Worker) {
-			eg.Go(func() error {
-				if policy := w.GCPolicy(); len(policy) > 0 {
-					return w.Prune(ctx, ch, policy...)
-				}
-				return nil
-			})
-		}(w)
+		eg.Go(func() error {
```

Comment:
I guess this is go1.22+ only

Reply:
Our `go.mod` says go1.22 so I just changed this while I was looking at it since this isn't needed anymore.